### PR TITLE
packages: add augeas

### PR DESCRIPTION
**Pull request by danlavu**

### CODE REVIEW

**Review by ikerexxe — COMMENTED** [review]
By reading this PR I have some general doubts about your objective:
* Where do you need to install this dependency? Client, or all images?
* Is this an integration test dependency?
* Did you forget Debian, or is this package installed by default in that distribution?

**Comment by danlavu** [comment]
> By reading this PR I have some general doubts about your objective:
> 
> * Where do you need to install this dependency? Client, or all images?
Client only
> * Is this an integration test dependency?
Yes, for authselect. 
> * Did you forget Debian, or is this package installed by default in that distribution?
I did forget, updated.

**Comment by ikerexxe** [comment]
> * Is this an integration test dependency?
>   Yes, for authselect.

The answer seems contradictory, at least to my knowledge. Let's set a common ground and check the question again.

Integration testing is a type of software testing where two or more modules of an application are logically grouped together and tested as a whole. System testing is types of testing where tester evaluates the whole system against the specified requirements.

IIUC, you'd like to test authselect as a whole against the specified requirements. In that case, you'd like to do system testing.

Source:
https://www.softwaretestinghelp.com/types-of-software-testing/

File: src/ansible/roles/packages/tasks/Fedora.yml [hunk 1]
```diff
@@ -206,6 +206,7 @@
     dnf:
       state: present
       name:
+      - augeas
```
Comment (ikerexxe) [inline src/ansible/roles/packages/tasks/Fedora.yml:209]: This should go in `Install test dependencies on client` task

File: src/ansible/roles/packages/tasks/Debian.yml [hunk 2]
```diff
@@ -55,6 +55,7 @@
       update_cache: yes
       name:
       - adcli
+      - augeas-tools
```
Comment (ikerexxe) [inline src/ansible/roles/packages/tasks/Debian.yml:58]: Create a new task called `Install test dependencies on client` after this task and add this dependency there.

File: src/ansible/roles/packages/tasks/Ubuntu.yml [hunk 3]
```diff
@@ -45,6 +45,7 @@
       update_cache: yes
       name:
       - adcli
+      - augeas-tools
```
Comment (ikerexxe) [inline src/ansible/roles/packages/tasks/Ubuntu.yml:48]: Create a new task called `Install test dependencies on client` after this task and add this dependency there.

**Review by ikerexxe — COMMENTED** [review]
Some minor comments and I think we'll be done

**Comment by danlavu** [comment]
Ack, all changes are done, yes, it is functional testing, not an integration test. Makes sense to create a new section.  ;)

**Review by ikerexxe — APPROVED** [review]
LGTM

**Comment by pbrezina** [comment]
Can you please change the commit message to `packages: add augeas`?

**Pull request merged**